pythoncz/templates/beginners_cs.html

```diff
@@ -163,7 +163,7 @@ <h2 id="onlineumim">Online materiály &ndash; programovat trochu umím</h2>
             Konvence pro psaní kódu
         </h3>
         <ul>
-            <li><a href="http://www.python.org/dev/peps/pep-0008/">Style Guide for Python Code</a></li>
+            <li><a href="http://pep8.org/">Style Guide for Python Code</a></li>
```

Comment:
Could you use HTTPS?

Reply:
I hope I did what you requested, please see the next commit.